app/templates/components/forms/user-payment-info-form.hbs

```diff
@@ -57,15 +57,10 @@
   <div class="field">
     <label >{{t 'Country'}}</label>
     <UiDropdown
-      @class="search selection"
+      @class="selection"
```

Comment:
We want it searchable

Reply:
Adding `search` would use a nested `<input>` tag which doesn't have `autocomplete="off"`. Is there a way to set that ?

Reply:
Yes, see Tax Info box in Create Event page